packages/plugin-core/src/survey.ts

```diff
@@ -223,4 +417,64 @@ export class SurveyUtils {
         });
     }
   }
+
+  static async showLapsedUserSurvey() {
+    AnalyticsUtils.track(SurveyEvents.InitialSurveyPrompted);
+    await vscode.window
+      .showInformationMessage(
+        "Could you share some feedback to help us improve?",
+        { modal: true },
+        { title: "Proceed" },
+        { title: "Skip Survey"},
```

Comment:
since the survey already has a cancel button, can we just use that?

Reply:
[this](https://github.com/dendronhq/dendron/pull/1409#issuecomment-927774529) is the reason I went with this, but the original intention was to _replace_ cancel with _Skip Survey_. I don't think we can remove the cancel button though. 

I'll try to figure something out.

Reply:
actually, just using the cancel button should be fine here.